patches/@unimodules+core+7.1.2.patch

```diff
@@ -1,319 +0,0 @@
-diff --git a/node_modules/@unimodules/core/.eslintrc.js b/node_modules/@unimodules/core/.eslintrc.js
```

Comment:
For my info:
Why is this removed?
What was it doing?
Why isn't it required anymore?

Reply:
because the package has been removed

Reply:
I am not too sure about this. Indeed it is still showing up in the yarn.lock file I see nothing that removes the package. It is being listed as an optional dep from isomorphic-webcrypto. 

So the question is. Where has this been explicitly removed and what is preventing its installation then?

Reply:
@nklomp this happened in sanders pull request